Tools/DumpRenderTree/mac/ResourceLoadDelegate.mm

```diff
@@ -263,7 +272,13 @@ -(void)webView: (WebView *)wv resource:identifier didFinishLoadingFromDataSource
 
 -(void)webView: (WebView *)wv resource:identifier didFailLoadingWithError:(NSError *)error fromDataSource:(WebDataSource *)dataSource
 {
-    if (!done && gTestRunner->dumpResourceLoadCallbacks()) {
+    if (done) {
```

Comment:
This code path should probably only be active if `!gUsingServerMode` is true, correct?

```suggestion
    if (!gUsingServerMode && done) {
```

Reply:
I was relying on `mainResourceURL` being `nil` but now that I've got a global variable accessible here, this is better. Thanks.